src/api/procedures/setVenueFiltering.ts

```diff
@@ -0,0 +1,62 @@
+import { Asset, PolymeshError, Procedure } from '~/internal';
+import { ErrorCode, SetVenueFilteringParams, TxTags } from '~/types';
+import { ExtrinsicParams, ProcedureAuthorization, TransactionSpec } from '~/types/internal';
+
+/**
+ * @hidden
+ */
+export type Params = SetVenueFilteringParams;
+
+/**
+ * @hidden
+ */
+export async function prepareVenueFiltering(
+  this: Procedure<Params, void>,
+  args: Params
+): Promise<TransactionSpec<void, ExtrinsicParams<'settlement', 'setVenueFiltering'>>> {
+  const {
+    context: {
+      polymeshApi: { tx, query },
+    },
+  } = this;
+
+  const { ticker, enabled: setEnabled } = args;
```

Comment:
```suggestion
  const { ticker, enabled } = args;
```
IMO there is no need for introducing a new variable here